.eslintrc.json

```
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "es2021": true,
+    "browser": true,
+    "mocha": true,
+    "node": true
+  },
+  "globals": {
+    "Promise": true,
+    "window": true,
+    "$": true,
+    "ga": true,
+    "jQuery": true,
+    "router": true
+  },
+  "settings": {
+    "react": {
+      "version": "16.4.2"
+    }
+  },
+  "extends": ["eslint:recommended", "plugin:react/recommended", "prettier"],
+  "overrides": [
+    {
+      "files": ["apps/secure-real-time-multiplayer-game/tests/1_unit-tests.js"],
+      "parserOptions": {
+        "sourceType": "module"
+      }
+    }
```

```suggestion
    },
    {
      "files": ["**/*.jsx"],
      "settings": {
        "react": {
          "version": "15.7.0"
        }
      },
      "extends": ["plugin:react/recommended"],
      "parserOptions": {
        "ecmaFeatures": {
          "jsx": true
        }
      }
    }
```
jsx files were getting ignored, but we can force eslint to pay attention via the `files` override. Also, not that it makes much difference, this is the React version we're using.